webpack/components/Table/TableHooks.js

```diff
@@ -36,9 +36,9 @@ class ReactConnectedSet extends Set {
 }
 
 const useSet = (initialArry) => {
-  const [, setToggle] = useState(false);
+  const [, setToggle] = useState(0);
   // needed because mutating a Ref won't cause React to rerender
-  const forceRender = () => setToggle(prev => !prev);
+  const forceRender = () => setToggle(prev => prev + 1);
```

Comment:
This was a great catch @sjha4 

Toggling the dummy state between `true` and `false` would result in a non-render if you had an even number of updates and React decided to batch them (e.g. true > false > true > false > true).  We never would have realized this bug if we'd had an odd number of environments. :+1:

Reply:
We could have a knowledge article saying never have even number of environments.. 😸